### PR TITLE
CreateEventScreen: create participants and setup park location search

### PR DESCRIPTION
**Pull request by petebrowne**
zogasjduvk

### CODE REVIEW

File: features/events/CreateEventScreen.js [hunk 1]
```diff
@@ -45,7 +46,15 @@ class CreateEventScreen extends Component {
...
   _create = async () => {
-    const {goBack, state} = this.props.navigation;
+    const {navigate, state} = this.props.navigation;
+    const {currentUser} = this.props.screenProps.auth;
+    const participations = this.state.invitees.map(i => ({
+      role: 'MEMBER',
+      userId: i,
+    }));
+    if (currentUser) {
+      participations.push({role: 'OWNER', userId: currentUser.id});
```
Comment (petebrowne) [inline features/events/CreateEventScreen.js:56]: I tried to auto-associate the current user as the "owner" of the event on Graph.cool but their hooks are pretty limited. This was much easier.

Comment (justinobney) [reply features/events/CreateEventScreen.js:56]: 👍

File: App.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ const authLink = setContext(async (_, {headers}) => {
 });
 
 const httpLink = new HttpLink({
-  uri: 'https://api.graph.cool/simple/v1/brecfast',
+  uri: 'http://localhost:60000/simple/v1/cje35wm3f000201398eo3q032',
```
Comment (justinobney) [inline App.js]: Was this supposed to get in?

**Review by justinobney — APPROVED** [review]

File: App.js (hunk 2 above)
Comment (petebrowne) [reply App.js]: Nope, what's the best way to setup ENV variables for local/production config?

**Pull request merged**